usaspending_api/database_scripts/matview_generator/chunked_matview_sql_generator.py

```diff
@@ -16,6 +18,10 @@
     make_matview_refresh,
     TEMPLATE,
 )
+from usaspending_api.etl.broker_etl_helpers import dictfetchall
```

Comment:
This line causes this script to fail because it tries to reference Django settings.
Usage in pipeline:
```
python3 usaspending_api/database_scripts/matview_generator/chunked_matview_sql_generator.py --file usaspending_api/database_scripts/matview_generator/transaction_search.json --chunk-count 10 --dest chunked_matviews
```

Reply:
Cool! The command works with the pipeline usage now.